static/css/drawer.css

```diff
@@ -237,6 +237,9 @@ h1, legend,
   height: 44px;
   line-height: 44px;
 }
+#buttonbar i.selected {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (i.selected) is overqualified, just use .selected without element name.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1563800563/issues/source?bid=2452379&fileBranchId=3019946#l240)